.changeset/tiny-actors-applaud.md

```diff
@@ -0,0 +1,7 @@
+---
+"@marigold/components": minor
+"@marigold/theme-preset": minor
+"@marigold/theme-b2b": major
```

Comment:
CC @lauraNawrotzki just to notify you adhead of time. If you are using any of the semantic tokens. With this PR they are now unified and all themes use the same tokens. And they will not change in the near future 🤞